workspaces/ui/src/components/diff/v2/CaptureManagerPage.js

```diff
@@ -428,6 +437,11 @@ function EndpointDiffs(props) {
                       className={classes.row}
                       component={Link}
                       to={to}
+                      onClick={() => {
+                        if (i.count > 0) {
+                          setTimeout(() => track("Viewing Endpoint Diff", i), 500)
```

Comment:
Can you explain the goal here?

Reply:
the timeout isn't necessary — that was something I had added because it wasn't working before, but I think now I do state management better so that's not an issue.

The if statement is so that the onClick action is only added to URL modals that are diffs (as technically a ListItem here can potentially have no diffs, and thus should not trigger any event upon a click. However, it seems that if there are no diffs, the onclick never gets triggered, so I don't think its needed